.pre-commit-config.yaml

```
@@ -0,0 +1,24 @@
+        entry: bash
+        language: system
+        files: \.bib$
+        args: [-c, 'bibtex -terse "$@" || true', --]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: trailing-whitespace
+        files: \.(tex|bib)$
+      - id: end-of-file-fixer
+        files: \.(tex|bib)$
+      - id: check-merge-conflict
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)